src/tox.ini

```diff
@@ -126,7 +126,7 @@ description =
     # W605: invalid escape sequence ‘x’
     # See https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
 deps = pycodestyle
-commands = pycodestyle --select E111,E211,E271,E306,E401,E502,E701,E702,E703,W291,W391,W605,E711,E712,E713,E721,E722 {posargs:{toxinidir}/sage/}
+commands = pycodestyle --select E111,E211,E271,E303,E306,E401,E502,E701,E702,E703,E714,W291,W293,W391,W605,E711,E712,E713,E721,E722 {posargs:{toxinidir}/sage/}
```

Comment:
As it says a few lines above in a comment, need to make the same change in `.vscode/settings.json`